src/services/visor-interactor.js

```diff
@@ -1,6 +1,22 @@
 function visorInteractor(textContent) {
 
 	console.log(textContent);
+const inputReuslt = document.getElementById("calculator-screen")
+inputReuslt.value = textContent
+/* 
+let lastValueTyped = inputReuslt.value[inputReuslt.value.length - 1];
```

Comment:
remove the code commented

Reply:
i know, i just wanted to see if i could use something of them